GoogleTestNuGet/Build.ps1

```diff
@@ -128,8 +128,8 @@ function Add-Signing {
     $xml = [xml](Get-Content "$Directory\$ProjectName.vcxproj")
 
     $MicroBuildProps = $xml.CreateElement("Import", "http://schemas.microsoft.com/developer/msbuild/2003")
-    $MicroBuildProps.SetAttribute("Project", "..\..\..\NuGetPackages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props")
-    $MicroBuildProps.SetAttribute("Condition", "Exists('..\..\..\NuGetPackages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props')")
+    $MicroBuildProps.SetAttribute("Project", "NuGetPackages\MicroBuild.Core.0.2.0\build\MicroBuild.Core.props")
```

Comment:
Didn't know about that! There might be one or two other places in here to set that as well, I'll make sure it works.